src/app/scenes/app.tsx

```diff
@@ -20,19 +22,23 @@ const setLanguage = useLanguage(
       ...require('@assets/i18n/en.json'),
```

Comment:
Please move this file into new directory `src/i18n/navigation`

Reply:
I decided to name this directory `scenes` because it can have translation not only for navigation, but also for screens